control/autoware_pure_pursuit/config/pure_pursuit.param.yaml

```diff
@@ -14,3 +14,4 @@
     curvature_calculation_distance: 4.0
     enable_path_smoothing: false
     path_filter_moving_ave_num: 25
+    control_period: 0.03
```

Comment:
Could you explain where you took this number from?
I think it's okay with 30Hz, but I just wanted to know if you had any reference for this value.

Reply:
@batuhanbeytekin frendly ping

Reply:
First of all, I apologize for my late response. I got this parameter value from [autoware_mpc_lateral_controller](https://github.com/autowarefoundation/autoware.universe/blob/main/control/autoware_mpc_lateral_controller/test/test_mpc.cpp#L100).